src/utils/image.ts

```diff
@@ -75,36 +76,39 @@ export function getDeviceIcon(info: DeviceInfo | SessionInfo) {
     }
 }
 
-export function getLibraryIcon(library: string | null | undefined) {
+export function getLibraryIcon(library: CollectionType | string | null | undefined) {
     switch (library) {
-        case 'movies':
-            return 'video_library';
-        case 'music':
-            return 'library_music';
-        case 'photos':
-            return 'photo_library';
-        case 'livetv':
+        case CollectionType.Movies:
+            return 'movie';
+        case CollectionType.Music:
+            return 'music_note';
+        case CollectionType.Homevideos:
+        case CollectionType.Photos:
+            return 'photo';
+        case CollectionType.Livetv:
             return 'live_tv';
-        case 'tvshows':
+        case CollectionType.Tvshows:
             return 'tv';
-        case 'trailers':
+        case CollectionType.Trailers:
             return 'local_movies';
-        case 'homevideos':
-            return 'photo_library';
-        case 'musicvideos':
+        case CollectionType.Musicvideos:
             return 'music_video';
-        case 'books':
-            return 'library_books';
+        case CollectionType.Books:
+            return 'book';
+        case CollectionType.Boxsets:
+            return 'video_library';
+        case CollectionType.Playlists:
+            return 'library_add';
```

Comment:
They look the same, but for consistency.
```suggestion
            return 'queue';
```